src/components/landing/pages/login/auth/via-email.tsx

```diff
@@ -39,24 +58,39 @@ export const ViaEMail: React.FC = () => {
 
           <Form.Group controlId="password">
             <Form.Control
-              isInvalid={error}
+              isInvalid={error !== EmailError.NONE}
               type="password"
               size="sm"
               placeholder={t('login.auth.password')}
               onChange={(event) => setPassword(event.currentTarget.value)}
               className="bg-dark text-white"/>
           </Form.Group>
 
-          <Alert className="small" show={error} variant="danger">
+          <Alert className="small" show={error === EmailError.LOGIN} variant="danger">
             <Trans i18nKey="login.auth.error.emailLogin"/>
           </Alert>
 
-          <Button
-            type="submit"
+          <Alert className="small" show={error === EmailError.REGISTER} variant="danger">
+            <Trans i18nKey="login.auth.error.emailRegister"/>
+          </Alert>
 
-            variant="primary">
-            <Trans i18nKey="login.signIn"/>
-          </Button>
+          <div className='flex flex-row email-buttons'>
```

Comment:
```suggestion
          <div className='flex flex-row email-buttons' dir='auto'>
```
shouldn't this be `dir='auto'`

Reply:
Added `dir='auto'` and removed unnecessary css-class `email-buttons`.